app/views/questionnaire.py

```diff
@@ -491,10 +491,20 @@ def _build_template(current_location, context=None, template=None):
 
     if previous_location is not None and not is_first_block_for_group and not current_location.block_id == 'thank-you':
         previous_url = previous_location.url(metadata)
+    elif current_location.group_id == 'who-lives-here-relationship':
```

Comment:
Will need to take is_first_block_for_group or current_location.block_id == 'thank-you' with me to do that, although I don't see a problem with that